git-tutorial.tex

```diff
@@ -100,6 +100,22 @@ \subsection{Commiting}
   \end{scriptsize}
 \end{frame}
 
+\subsection{Commit messages}
+\begin{frame}{Commit messages}
+  \begin{itemize}
+    \item What's an \textbf{in-depth explanation}? The items below aren't an exhaustive list, but should be considered when writing;
```

Comment:
IMO, this should out of the `itemize` block.

I haven't checked if this a pattern used elsewhere, but it could be inside a `block` with a header. Something like this:

```latex
\begin{block}{What's an \textbf{in-depth explanation?}}
  \begin{itemize}
    ...
  \end{itemize}
\end{block}
```